admin2.php

```diff
@@ -22,46 +22,86 @@
 	</head>
 	
 	<body>
-		
+	
 		<!-- grid-container class which contains all content using CSS grid -->
 		<div class="grid-container">
 			
-			<!-- navigation located at the very top -->				
+			<!-- navigation located at the very top -->	
 			<div class="nav">
 				<b><p style = "color: white; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Navigation</p></b>
 			</div>
+			
+			<!-- login box -->
+			<div class = "login_box">
 
-			<!-- content_1 to hold the header and login box -->
-			<div class = "content_1">
-				
-				<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Header</p></b>
+				<b><p style = "color: black; font-size: 2vw; margin: 5.5vw 0vw 0vw 3.7vw;">Update Username</p></b>
+
+				<!-- Form for input fields -->
+				<h3><form method = "post" id= "01_login">
+
+					<p style = "margin: -2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Existing Username:</p>
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "ExistingUserName" placeholder="Your Existing Username..."/>
+
+					<p style = "margin: 1.5vw 0vw 0vw 3.7vw; font-size: 1.4vw;">New Username</p>
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "NewUserName" placeholder="Your New Username..."/>
+
+					<input style = "width: 19.2vw; height: 2.5vw; margin: 2.8vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "submit" name = "submit" value = "Update Username"/>
 				
-				<!-- login box -->
-				<div class = "login_box">
+				</form></h3>
+
+				<center><p style = "margin: -5.8vw 0vw 0vw 0vw; font-size: 1vw; visibility: hidden;">Return Back To<a href = "login.php"> Login</a></p></center>
+
+			</div>
+
+			<!-- content_1 to hold header, and content_2 -->
+			<div class = "content_1">
+			
+				<!-- header class which consists of the header text -->
+				<div class = "header">
+					<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw; ">Header</p></b>
+				</div>
+
+				<!-- content_2, which contains content_3 -->
+				<div class = "content_2" style = "background-color: transparent; height: 17vw;">
 				
-					<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Update Username</p></b>
-					
-					<h3><form method = "post" id= "01_login">
-						
-						<p style = "margin: 2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Existing Username:</p>
-						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "ExistingUserName" placeholder="Your Existing Username..."/>
+					<!-- content_3, which contains the query row title and query row output via SQL -->
+					<div class = "content_3" style = "background-color: transparent;">

```

Comment:
Added content_3, which contains the display users query title row and query output box.